base/healthcheck-docker/Dockerfile

```diff
@@ -14,14 +14,12 @@
 FROM gcr.io/kctf-docker/kctf-pwntools:latest AS pwntools
 FROM ubuntu:19.10
 
-RUN apt-get update && apt-get upgrade -y
```

Comment:
removing upgrade commands, and instead using `apt-get update && apt-get install` so that we always install latest.